File: problems/j/jumbojavelin/jumbojavelin.c
/*
Author: Rudi César Comiotto Modena
Email: rudi.modena@gmail.com
*/

#include <stdio.h>

int main(void) {
    int n, l, length;
    if (scanf("%d", &n) != 1) {
        printf("Failed to read n!");
    }

    length = 0;
    for (int i=0; i<n; i++) {
        if (scanf("%d", &l) != 1) {
            printf("Failed to read n!");
        }
        length += l;
    }

    length -= n - 1;

    printf("%d", length);

    return 0;
}

File: problems/j/jumbojavelin/jumbojavelin.py
#! /usr/bin/python3
"""
Author: Rudi César Comiotto Modena
Email: rudi.modena@gmail.com
"""

from re import L


def main():
    n = int(input())

    length = 0
    for _ in range(n):
        l = int(input())
        length += l

    print(length - (n - 1))


if __name__ == '__main__':
    main()